build/vsts_build.yaml

```diff
@@ -28,8 +28,10 @@ phases:
         $newBuildCounter = [System.Decimal]::Parse($revision)
         $newBuildCounter++
         Set-Content $env:BUILDCOUNTERFILE $newBuildCounter
-        Write-Host "##vso[build.updatebuildnumber]$newBuildCounter"
+        $FullBuildNumber = "4.8.0.$newBuildCounter"
```

Comment:
Can we also configure `4.8.0` part with variable? or could use it from common config?

Reply:
good suggestion, done.